src/Control/Lens/Setter.hs

```diff
@@ -91,11 +91,12 @@ import Control.Monad.State.Class  as State
 import Control.Monad.Writer.Class as Writer
 import Data.Functor.Contravariant
 import Data.Functor.Identity
-import Data.Monoid
+import Data.Monoid (Monoid (..))
```

Comment:
Guard this behind `#if !(MIN_VERSION_base(4,8,0))` to avoid redundant import warnings on recent GHCs.